extract_unique_sensors_01.py

```diff
@@ -0,0 +1,40 @@
+from urllib.request import urlopen
+from pathlib import Path
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    path = Path(url)
+    filename = path.name
+    with open(filename) as lines:
+        sensors_data = lines.readlines()
+    
+def extract_all_sensors(lines): 
+    """Filters the sensors_data and returns only a list of sensors. The function accepts lines
+    as a string and nothing else.
+    """
+    assert isinstance(lines, list), "Expected a list."
+    candidate_lines = []
+    for line in lines:
+        if line.startswith('kat.sensors.'):
+            for candidate_line in line.split():
+                candidate_lines.append(candidate_line)
+    return candidate_lines
+
+# Loop over candidate_lines and extract sensor names
+def splitting_list():
+    """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
+    empty lines.
+    """
+    sensors_list = []
+    candidate_lines = extract_all_sensors(sensors_data)
+    for sensor in candidate_lines:
+    # Typical output from "sensor" variable:
+    # kat.sensors.anc_api_version   kat.sensors.anc_ganglia_api_version   kat.sensors.anc_ganglia_kat_monctl_system_os_name
+        sensor = sensor.split('.')[-1]  
+        sensor = sensor.strip("\n") 
+        sensors_list.append(sensor)
+    return sensors_list 
+print(sorted(splitting_list()))
```

Comment:
Call the print in the `if __name__` condition.

example:
```python
if __name__ == "__main__":
    print(sorted(....
```

Read: 
- https://stackoverflow.com/questions/419163/what-does-if-name-main-do
- https://www.journaldev.com/17752/python-main-function